.gitattributes

```diff
@@ -0,0 +1,5 @@
+*.rb eol=lf
```

Comment:
Please do a rebase against the master branch, because this file is managed by ```voxpupuli/modulesync```, ```voxpupuli/modulesync_config```.